manual/src/refman/extensions/effects.etex

```diff
@@ -1,13 +1,5 @@
 (Introduced in 5.0)
 
-\textit{Note: Effect handlers in OCaml 5.0 should be considered experimental.
```

Comment:
I would add the fact that the syntax was added in `5.3`:

 (Introduced in 5.0, effect handler syntax added in 5.3)

Reply:
Done in https://github.com/ocaml/ocaml/pull/13295/commits/afd618f0c42a3cccb90ef461d213237f598030bc.